dask/array/utils.py

```diff
@@ -276,3 +276,4 @@ def __array_function__(self, *args, **kwargs):
 
 
 IS_NEP18_ACTIVE = _is_nep18_active()
+NDARRAY_ARRAY_FUNCTION = getattr(np.ndarray, '__array_function__', None)
```

Comment:
Why this change?  Was the getattr call expensive in some way?  If not, I'd rather inline this and avoid hiding this.

Reply:
Inlined. I didn't want there to be a `getattr` call in a loop. 😄